runtime-decompiler/src/main/java/org/jrd/backend/decompiling/PluginManager.java

```diff
@@ -238,8 +240,17 @@ public void deleteWrapper(DecompilerWrapperInformation wrapperInformation) {
         wrappers.remove(wrapperInformation);
 
         if (wrapperInformation.getScope().equals("local")) {
-            new File(wrapperInformation.getFileLocation()).delete();
-            new File(flipWrapperExtension(wrapperInformation.getFileLocation())).delete();
+            try {
+                Files.delete(Path.of(wrapperInformation.getFileLocation()));
+            } catch (IOException e) {
+                OutputController.getLogger().log(OutputController.Level.MESSAGE_ALL, e);
+            }
+
+            try {
+                Files.delete(Path.of(flipWrapperExtension(wrapperInformation.getFileLocation())));
+            } catch (IOException e) {
+                OutputController.getLogger().log(OutputController.Level.MESSAGE_ALL, e);
+            }
         }
```

Comment:
What was the case when those were failing?
Isnt better to not delete, if there is nothing to delete?

Reply:
The case was that the file was being used by another process (or rather, another thread of JRD - see 29a4107d).

> Isnt better to not delete, if there is nothing to delete?
This doesn't delete if there is nothing to delete either? It just allows for an explanation why the deletion failed via an Exception, rather than the original (and ignored!) boolean return value.